### DISCUSSION TITLE
Feedback on Tailwind Elements

### DISCUSSION THREAD

**Discussion by jiorpilla** [mdbootstrap/TW-Elements#1707] [Kind words & general feedback]
Hello Tailwind Elements team!

I just started using Tailwind Elements and I wanted to give some feedback.
Overall, I think it's a fantastic product and it's really helped me *speed up my web development process*.

I'm a beginner in NPM modules and I just want to confirm, is directly accessing JavaScript like this <script type="text/javascript" src="../node_modules/tw-elements/dist/js/tw-elements.umd.min.js"></script> okay? or are we recommended to copy/move the file once we deploy our finished website. is it not possible to have a treeshaking without vite?

Shout out to [Michal](https://twitter.com/ascensus_mdb) & [Dawid](https://twitter.com/dawidadach) for starting this & great thanks to @Trochonovitz @juujisai @Martussky @smolenski-mikolaj for all the work put in the latest release!

Keep coding!

**Comment by juujisai** [comment]
Hi, when deploying your website, it's best to avoid directly accessing JavaScript files from the node_modules folder using a script tag. Instead, copy or move the necessary JavaScript files to a designated folder within your project. This ensures proper organization and accessibility. Treeshaking can be achieved with tools like Vite, Webpack, or Rollup.

**Reply by jiorpilla** [reply]
Thank you @juujisai, 
I have integrated TE with my project using Webpack.
